.storybook/main.ts

```diff
@@ -13,7 +13,6 @@ const config: StorybookConfig = {
     '@storybook/addon-essentials',
     '@storybook/addon-onboarding',
     '@storybook/addon-interactions',
-    '@storybook/addon-styling',
```

Comment:
The Package @storybook/addon-styling has been deprecated : mostly used for dark theme option , removed it currently . Its been replaced by @storybook/addon-themes but that’s in storybook v9